instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/tapir/RouteWrapper.java

```diff
@@ -19,6 +23,7 @@
 import sttp.tapir.server.ServerEndpoint;
 
 public class RouteWrapper implements Function1<RequestContext, Future<RouteResult>> {
+  private static final Uri.Path EMPTY = Uri.Path$.MODULE$.apply("", Charset.defaultCharset());
```

Comment:
This seems to be the only way to get the `Uri.Path.Empty` object from java - apparently once it's 3 or more levels deep, accessing nested scala classes from java gets tricky.